lib/tictactoe.rb

```diff
@@ -1,6 +1,17 @@
 class Tic_Tac_Toe
   def run_app()
     puts("Welcome to Tic-Tac-Toe")
-    "Welcome to Tic-Tac-Toe"
   end
-end
\ No newline at end of file
+
+  def show_empty_board
```

Comment:
Let's change this to return a `string` representation of the board, so that our test doesn't need to look at `stdout`

Reply:
👍🏽